app/assets/javascripts/discourse/app/controllers/discovery-sortable.js

```diff
@@ -13,6 +13,7 @@ export const queryParams = {
   before: { replace: true, refreshModel: true },
   bumped_before: { replace: true, refreshModel: true },
   f: { replace: true, refreshModel: true },
+  s: { replace: true, refreshModel: true },
```

Comment:
I think we should avoid a single letter query params going forward. They provide no benefits while having a huge impact on readability.